components/homepage/NewsFeed.js

```diff
@@ -87,7 +90,7 @@ class NewsFeed extends React.Component {
                       <h3>{item.title}</h3>
                     </a>
                     {item.date && <div className="news__item__date">Published on {item.date}</div>}
-                    {item.description && <p>{item.description}</p>}
+                    {item.description && <Parser>{md.render(item.description)}</Parser>}
```

Comment:
Nothing on the website for frontmatter-markdown-loader actually says that it will process markdown in the yaml attributes, so I used the markdown parser that frontmatter-markdown-loader uses directly.

Reply:
Do you think it's a smell that we're including Markdown in the YAML attributes? From the docs for [frontmatter-markdown-loader](https://www.npmjs.com/package/frontmatter-markdown-loader) and [the examples of Netlify CMS](https://www.netlifycms.org/docs/examples/), the Markdown content goes beneath the YAML frontmatter.

Reply:
@nholden that's exactly what I found when I was looking into this initially as well - they only expect markdown beneath the YAML. The way we have the homepage news feed configured in Netlify is to use the [list widget](https://www.netlifycms.org/docs/widgets/list/#list), and there is no way to save data from that widget outside of YAML (as far as I know).

The alternative (no-smelly) way to do this would be to configure Netlify to have a "post" collection. That would allow us to save full body content as markdown beneath the frontmatter.

It may be something to revisit in the future if we don't want to keep parsing it as @haywood has done.

Reply:
Yeah, post collection definitely sounds more correct :) I'll make a post-launch issue for that

Reply:
https://github.com/texas-justice-initiative/website-nextjs/issues/160